spec/models/note_spec.rb

```diff
@@ -1,5 +1,170 @@
 require 'rails_helper'
 
-RSpec.describe Note, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+shared_examples 'a valid note' do
+  it 'passes the content_length validation' do
+    note = described_class.new(note_type: note_type, utility: utility)
+    note.validate_content_length
+    expect(note.errors).to be_empty
+  end
+end
+
+shared_examples 'an invalid note' do
+  it 'doesnt pass the content_length validation' do
+    note = described_class.new(note_type: note_type, utility: utility)
+    note.validate_content_length
+    expect(note.errors).not_to be_empty
+  end
+end
+
+shared_examples 'content length' do
+  it 'is expected' do
+    note = build(:note, content: Faker::Lorem.sentence(word_count: word_count), utility: utility)
+    expect(note.content_length).to eq(expected)
+  end
+end
+
+shared_examples 'content length cases for utility' do |word_count, expected|
+  context "when content is #{expected}" do
+    let(:word_count) { word_count }
+    let(:expected) { expected }
+
+    it_behaves_like 'content length'
+  end
+end
+
+shared_examples 'validates review content length' do
+  context 'when short' do
+    before do
+      allow_any_instance_of(described_class).to receive(:content_length).and_return('short')
+    end
+
+    it_behaves_like 'a valid note'
+  end
+
+  context 'when medium' do
+    before do
+      allow_any_instance_of(described_class).to receive(:content_length).and_return('medium')
+    end
+
+    it_behaves_like 'an invalid note'
+  end
+
+  context 'when long' do
+    before do
+      allow_any_instance_of(described_class).to receive(:content_length).and_return('long')
+    end
+
+    it_behaves_like 'an invalid note'
+  end
+end
+
+shared_examples 'validates critique content length' do
+  context 'when short' do
+    before do
+      allow_any_instance_of(described_class).to receive(:content_length).and_return('short')
+    end
+
+    it_behaves_like 'a valid note'
+  end
+
+  context 'when medium' do
+    before do
+      allow_any_instance_of(described_class).to receive(:content_length).and_return('medium')
+    end
+
+    it_behaves_like 'a valid note'
+  end
+
+  context 'when long' do
+    before do
+      allow_any_instance_of(described_class).to receive(:content_length).and_return('long')
+    end
+
+    it_behaves_like 'a valid note'
+  end
```

Comment:
Acá si bien estamos probando casos exhaustivos, veo mucha repetición, tal vez podemos resumirlo tal vez de la siguiente manera:
```suggestion
  context 'when the critique length is valid' do
    let(:valid_lengths) { %w[short medium length] }
  
    before do
      allow_any_instance_of(described_class).to receive(:content_length).and_return(valid_lengths.sample)
    end

    it_behaves_like 'a valid note'
  end
```
Ya que en este caso cualqueir longitud nos da una crítica válida